lib/workers/repository/process/fetch.ts

```diff
@@ -42,11 +44,25 @@ async function fetchDepUpdates(
     dep.skipReason = 'disabled';
   } else {
     if (depConfig.datasource) {
-      dep = {
-        ...dep,
-        ...(await lookupUpdates(depConfig as LookupUpdateConfig)),
-      };
+      try {
+        dep = {
+          ...dep,
+          ...(await lookupUpdates(depConfig as LookupUpdateConfig)),
+        };
+      } catch (err) {
+        if (config.repoIsOnboarded || !(err instanceof ExternalHostError)) {
```

Comment:
i think you can use `packageFileConfig` here instead?

Reply:
Good point, thanks 👍